### PR TITLE
Overhaul events

### LINKED ISSUE #291: Overhaul Events
Currently the namings of events in various xpallets are inconsistent, basically we should follow the rule of Substrate:

- app-friendly event description(https://github.com/paritytech/substrate/pull/6684)
- better to use passive voice(https://github.com/paritytech/substrate/pull/6960#discussion_r479937473)

### PR DESCRIPTION
**Pull request by liuchengxu**
- New chainx_types. cc @qinghuan-chain 
- A bunch of events have been refactored, mostly about the naming. cc @tarvos21

Close #291

### CODE REVIEW

File: xpallets/gateway/records/src/lib.rs [hunk 1]
```diff
@@ -75,11 +75,17 @@ decl_event!(
         Balance = BalanceOf<T>,
         WithdrawalRecord = WithdrawalRecordOf<T>
     {
+        /// An account deposited some asset. [who, asset_id, amount]
         Deposited(AccountId, AssetId, Balance),
-        WithdrawalApplied(WithdrawalRecordId, WithdrawalRecord),
+        /// A withdrawl application was created. [withdrawal_id, record_info]
+        WithdrawalApplicationCreated(WithdrawalRecordId, WithdrawalRecord),
```
Comment (koushiro) [inline xpallets/gateway/records/src/lib.rs]:
```suggestion
        WithdrawalCreated(WithdrawalRecordId, WithdrawalRecord),
```

File: xpallets/system/src/lib.rs [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use sp_runtime::traits::StaticLookup;
```
Comment (koushiro) [inline xpallets/system/src/lib.rs]: I prefer to place the part of the standard library separately at the top of the imports

File: xpallets/system/src/lib.rs [hunk 3]
```diff
@@ -62,8 +62,8 @@ decl_storage! {
         /// Paused pallet call.
         pub Paused get(fn paused): map hasher(twox_64_concat) Vec<u8> => BTreeMap<Vec<u8>, ()>;
 
-        /// Blocked account ids.
-        pub BlockedAccounts get(fn blocked_accounts): map hasher(blake2_128_concat) T::AccountId => bool;
+        /// The accounts that are blocked.
+        pub Blacklist get(fn blacklist): map hasher(blake2_128_concat) T::AccountId => bool;
```
Comment (koushiro) [inline xpallets/system/src/lib.rs:67]: I prefer to use `Blocklist` instead of `Blacklist`

File: xpallets/gateway/bitcoin/src/lib.rs [hunk 4]
```diff
@@ -166,28 +166,28 @@ decl_event!(
...
+        /// A unclaimed deposit record was removed. [depositor, deposit_amount, tx_hash, chain_addr]
+        PendingDepositRemoved(AccountId, Balance, H256, AddrStr),
+        /// A new withdrawal proposal was created. [proposer, withdrawal_ids]
+        ProposalCreated(AccountId, Vec<u32>),
+        /// A trustee voted/vetoed a withdrawal proposal. [trustee, vote_status]
+        ProposalVoted(AccountId, bool),
+        /// A withdrawal proposal was dropped. [reject_count, total_count, withdrawal_ids]
+        ProposalDropped(u32, u32, Vec<u32>),
+        /// The proposal has been processed successfully and is waiting for broadcasting. [tx_hash]
+        ProposalCompleted(H256),
```
Comment (koushiro) [inline xpallets/gateway/bitcoin/src/lib.rs]:
```suggestion
        WithdrawalProposalCreated(AccountId, Vec<u32>),
        /// A trustee voted/vetoed a withdrawal proposal. [trustee, vote_status]
        WithdrawalProposalVoted(AccountId, bool),
        /// A withdrawal proposal was dropped. [reject_count, total_count, withdrawal_ids]
        WithdrawalProposalDropped(u32, u32, Vec<u32>),
        /// The proposal has been processed successfully and is waiting for broadcasting. [tx_hash]
        WithdrawalProposalCompleted(H256),
```

**Review by atenjin — APPROVED** [review]

**Review by koushiro — COMMENTED** [review]
Rest LGTM

File: xpallets/system/src/lib.rs (hunk 3 above)
Comment (liuchengxu) [reply xpallets/system/src/lib.rs:67]: Since it's already used in Substrate and `Blocklist` is not a _valid_ word in the dictionary, I'll keep it for now.

**Pull request merged**